Gemfile

```diff
@@ -13,7 +13,7 @@ gem "unicorn-worker-killer", "0.4.4"
 
 # Federation
 
-gem "diaspora_federation-rails", "0.1.8"
+gem "diaspora_federation-rails", github: "SuperTux88/diaspora_federation", branch: "protocol-next-step"
```

Comment:
Note: Replace this with a replaced version.

(I know that's your plan, I just want to have a note below this line so it's marked as addressed when it is.)